src/index.html

```diff
@@ -17,16 +17,31 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
-
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search-form search-form_big"
+      data-qa="big"
+    >
+      <label class="search-form_label search-form_label_big">
+        <input
+          type="text"
+          class="search-form_input search-form_input_big"
+          data-qa="keypress"
+          placeholder="Try “Los Angeles“"
+        />
+      </label>
+    </form>
+    <form
+      class="search-form search-form_small"
+      data-qa="small"
+    >
```

Comment:
Similar to the previous form, this `<form>` tag is self-closing, which is not valid in HTML. Change `<form />` to `<form>` and ensure it has a corresponding `</form>`.